moneyed/test_moneyed_classes.py

```diff
@@ -57,6 +57,12 @@ def test_init(self):
     def test_repr(self):
         assert str(self.default_curr) == self.default_curr_code
 
+    def test_hash(self):
+        try:
+            set([self.default_curr])
+        except TypeError:
+            assert False, "%s is not hashable" % type(self.default_curr)
+
```

Comment:
A simpler and more helpful test would have been:

    assert self.default_curr in set([self.default_curr])

This is less code, and if there is an exception, it doesn't hide what the exception is. The test above (at least under Python 2) will translate an informative error into a less informative one